compose/compose.go

```diff
@@ -158,10 +164,17 @@ func getOffset(idx int) int {
 	return idx
 }
 
-func initService(basename string, idx, grpcPort int) service {
+func initService(basename string, idx, grpcPort int, container_name string) service {
```

Comment:
In Go, camelcase is the idiomatic way for defining variables. Please don't use camelcase.

Reply:
okay.